python/tk_multi_launchapp/single_config_launcher.py

```diff
@@ -74,10 +74,19 @@ def register_launch_commands(self):
         # Initialize per version
         app_versions = self._tk_app.get_setting("versions") or []
         if app_versions:
+            # Information used to construct the equivalent of Desktop "collapse rules"
```

Comment:
I think for this change, we don't want to change any of the current behaviour, only the new behaviour when you use the software entity. For the old config based launch entries, we just don't do any attempt to group things.